packages/atomic/src/components/product-listing/index.ts

```diff
@@ -0,0 +1,65 @@
+  buildFacet as buildProductListingFacet,
+  FacetOptions as ProductListingFacetOptions,
+  FacetValue as ProductListingFacetValue,
+  NumericRangeRequest as ProductListingNumericRangeRequest,
+  NumericFacet as ProductListingNumericFacet,
+  DateFacet as ProductListingDateFacet,
+  NumericFacetValue as ProductListingNumericFacetValue,
+  NumericFacetState as ProductListingNumericFacetState,
+  NumericFilter as ProductListingNumericFilter,
+  NumericFilterState as ProductListingNumericFilterState,
+  DateFacetValue as ProductListingDateFacetValue,
+  DateFacetState as ProductListingDateFacetState,
+  DateFilter as ProductListingDateFilter,
+  DateFilterState as ProductListingDateFilterState,
+  DateFacetOptions as ProductListingDateFacetOptions,
+  DateRangeOptions as ProductListingDateRangeOptions,
+  DateRangeRequest as ProductListingDateRangeRequest,
+  buildDateRange as buildProductListingDateRange,
+  buildDateFacet as buildProductListingDateFacet,
+  buildDateFilter as buildProductListingDateFilter,
+  loadDateFacetSetActions as loadProductListingDateFacetSetActions,
+  buildNumericFacet as buildProductListingNumericFacet,
+  buildNumericFilter as buildProductListingNumericFilter,
+  buildNumericRange as buildProductListingNumericRange,
+  loadNumericFacetSetActions as loadProductListingNumericFacetSetActions,
+  FacetManager as ProductListingFacetManager,
+  buildFacetManager as buildProductListingFacetManager,
+  Pager as ProductListingPager,
+  PagerState as ProductListingPagerState,
+  buildPager as buildProductListingPager,
+  buildResultsPerPage as buildProductListingResultsPerPage,
+  buildInteractiveResult as buildProductListingInteractiveResult,
+  InteractiveResult as ProductListingInteractiveResult,
+} from '@coveo/headless/product-listing';
+
+export {
+  RelativeDate as ProductListingRelativeDate,
+  deserializeRelativeDate as deserializeProductListingRelativeDate,
+  FacetSortCriterion as ProductListingFacetSortCriterion,
+  FacetValueRequest as ProductListingFacetValueRequest,
+  CategoryFacetValueRequest as ProductListingCategoryFacetValueRequest,
+  FacetConditionsManager as ProductListingFacetConditionsManager,
+  RangeFacetSortCriterion as ProductListingRangeFacetSortCriterion,
+  RangeFacetRangeAlgorithm as ProductListingRangeFacetRangeAlgorithm,
+  AnyFacetValuesCondition as ProductListingAnyFacetValuesCondition,
+  buildFacetConditionsManager as buildProductListingFacetConditionsManager,
+  Tab as ProductListingTab,
+  TabState as ProductListingTabState,
+  buildTab as buildProductListingTab,
+  BreadcrumbManagerState as ProductListingBreadcrumbManagerState,
+  BreadcrumbManager as ProductListingBreadcrumbManager,
+  buildBreadcrumbManager as buildProductListingBreadcrumbManager,
+  loadQuerySetActions as loadProductListingQuerySetActions,
+  getOrganizationEndpoints as getOrganizationEndpointsProductListing,
+  PlatformEnvironment as PlatformEnvironmentProductListing,
+} from '@coveo/headless';
```

Comment:
We will need to add these to `@coveo/headless/product-listing` to avoid exporting from 2 different packages.

Reply:
👍